opentelemetry-collector-contrib.yaml

```diff
@@ -12,8 +12,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - go
```

Comment:
The whole environment paragraph can be deleted. All of the needed things here are pulled in by go/build pipeline already.

Reply:
Thank you for your thorough reviews @xnox -- fixed!!